README.md

```diff
@@ -565,16 +536,6 @@ Get Wire
 The `wire-runtime` package contains runtime support libraries that must be included in applications
 that use Wire-generated code.
 
-With Maven:
```

Comment:
This should still be valid.

Reply:
True.  Should we even mention it though?

Reply:
Don't have a strong opinion.

Reply:
Let's keep it IMO